test/unit-tests/presets.test.ts

```diff
@@ -1,13 +1,18 @@
 import { Condition, evaluateCondition, getArchitecture, getToolset } from '../../src/preset';
 import { expect } from '@test/util';
+import * as os from "os";
 
 suite('Preset tests', () => {
     test('Parse architecture', () => {
         expect(getArchitecture({ name: 'test', architecture: 'x86' })).to.eq('x86');
         expect(getArchitecture({ name: 'test', architecture: 'amd64' })).to.eq('amd64');
         expect(getArchitecture({ name: 'test', architecture: { value: 'arm', strategy: 'set' } })).to.eq('arm');
         expect(getArchitecture({ name: 'test', architecture: { value: 'arm64', strategy: 'external' } })).to.eq('arm64');
-        expect(getArchitecture({ name: 'test' })).to.eq('x86');
+        if (os.platform() === "win32" || os.platform() === "linux") {
+            expect(getArchitecture({ name: "test" })).to.eq("x64");
```

Comment:
is it verified that os.platform on windows arm64 is not also win32? 

asking because this is what i found in the docs:
process.platform[#](https://nodejs.org/dist/latest-v4.x/docs/api/process.html#process_process_platform)

What platform you're running on: 'darwin', 'freebsd', 'linux', 'sunos' or 'win32'

Reply:
Fixed to be more explicit